internal/witness/cmd/witness/Dockerfile

```diff
@@ -19,7 +19,7 @@ COPY . .
 RUN go build -o /build/bin/witness ./internal/witness/cmd/witness
 
 # Build release image
-FROM golang:buster
+FROM golang:buster@sha256:fd9306e1c664bd49a11d4a4a04e41303430e069e437d137876e9290a555e06fb
```

Comment:
Is it expected that this is a different hash to the `golang:buster` above?

Reply:
Good catch. The sha256 hash was provided in the code scanning alerts. Updated to use the image created 2 days ago.

```
➜  ~ docker images --digests
REPOSITORY    TAG       DIGEST                                                                    IMAGE ID       CREATED         SIZE
golang        <none>    sha256:fd9306e1c664bd49a11d4a4a04e41303430e069e437d137876e9290a555e06fb   d5a118f29bfa   2 days ago      845MB
golang        <none>    sha256:eb3f9ac805435c1b2c965d63ce460988e1000058e1f67881324746362baf9572   c94784fe2595   3 weeks ago     719MB
```